cl/citations/utils.py

```diff
@@ -44,3 +50,32 @@ async def get_citation_depth_between_clusters(
         cited_opinion__cluster__pk=cited_cluster_pk,
     ).aaggregate(depth=Sum("depth"))
     return result["depth"]
+
+
+def get_years_from_reporter(
+    citation: FullCaseCitation,
+) -> tuple[int, int]:
+    """Given a citation object, try to look it its dates in the reporter DB"""
+    start_year = 1750
+    end_year = date.today().year
+
+    edition_guess = citation.edition_guess
+    if edition_guess:
+        if hasattr(edition_guess.start, "year"):
+            start_year = edition_guess.start.year
+        if hasattr(edition_guess.end, "year"):
+            start_year = edition_guess.end.year
+    return start_year, end_year
+
+
+def make_name_param(
+    defendant: str,
+    plaintiff: str | None = None,
+) -> tuple[str, int]:
+    """Remove punctuation and return cleaned string plus its length in tokens."""
+    token_list = defendant.split()
+    if plaintiff:
+        token_list.extend(plaintiff.split())
+        # Strip out punctuation, which Solr doesn't like
```

Comment:
I think that this comment has incorrect indentation

Reply:
yeah, fixed it!